data/data/openstack/main.tf

```diff
@@ -72,3 +72,12 @@ module "topology" {
   trunk_support       = var.openstack_trunk_support
   octavia_support     = var.openstack_octavia_support
 }
+
+resource "openstack_images_image_v2" "base_image" {
+  name             = var.openstack_base_image
```

Comment:
This should be a generated ID

Reply:
Yes, this is what's breaking the OpenStack CI.

On the subject of naming: we should at least add the same ID prefix to the image name as all the other resources we create.  See `${var.cluster_id}-` here:

https://github.com/openshift/installer/blob/10975dcbf96c488184d3bfc2cfbbd059fd295e78/data/data/openstack/topology/private-network.tf#L12

That should solve the immediate issue and align the image name with everything else.

That said, I'd prefer if we could also put the actual image version in the name too (rather than just "<prefix>-rhcos". Is there any chance we could get that from the URL? I'm guessing that would make things way more complicated so it's probably not worth the effort in this PR, but if you can think of a good way, that would be awesome.